cmd/monaco/continue_deployment_on_error_integration_test.go

```diff
@@ -0,0 +1,64 @@
+// +build integration
+
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/api"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/environment"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/project"
+	"os"
+	"testing"
+
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/util"
+	"gotest.tools/assert"
+)
+
+// tests all configs for a single environment
+func TestIntegrationContinueDeploymentOnError(t *testing.T) {
+
+	const allConfigsFolder = "test-resources/integration-configs-with-errors/"
+	const allConfigsEnvironmentsFile = allConfigsFolder + "environments.yaml"
+
+	RunIntegrationWithCleanup(t, allConfigsFolder, allConfigsEnvironmentsFile, "AllConfigs", func(fileReader util.FileReader) {
+
+		environments, errs := environment.LoadEnvironmentList("", allConfigsEnvironmentsFile, fileReader)
+		assert.Check(t, len(errs) == 0, "didn't expect errors loading test environments")
+
+		projects, err := project.LoadProjectsToDeploy("", api.NewApis(), allConfigsFolder, fileReader)
+		assert.NilError(t, err)
+
+		os.Setenv("NEW_CLI", "1")
```

Comment:
Maybe we should put the switch what cli to use in the `RunImpl` 
method definition. That way we do not need to set env variables.

Reply:
Yes, talked also to @didiladi and we should actually run tests with both cli versions. Having a switch would be helpful.

Reply:
Leaving this as it s for now, we anyways need a good approach in running tests on both CLIs. We'll have discussion about this on the roadmap planning.